extension/src/test/fixtures/plotsDiff/index.ts

```diff
@@ -809,9 +821,7 @@ export const getComparisonWebviewMessage = (
       ? getImageDataWithMultiImgs(baseUrl, joinFunc)
       : getImageData(baseUrl, joinFunc)
   )) {
-    const multiImagePath = joinFunc('plots', 'image')
-    const isMulti = path.includes(multiImagePath)
-    const pathLabel = path
+    const pathLabel = path.includes('image') ? join('plots', 'image') : path
```

Comment:
We had to use `join` here so the tests would pass on windows (we rebuild the `image` path when collecting the selected plots).